numba/cuda/tests/cudapy/test_compiler.py

```diff
@@ -61,7 +61,7 @@ def f(x, y, z, d):
         # With fastmath, ftz and approximate div / sqrt are enabled
         self.assertIn('fma.rn.ftz.f32', ptx)
         # "full" refers to a full-range approximate divide
```

Comment:
Is this comment still applicable?

Reply:
Apparently not. `full` isn't seen in the ptx any more.